src/main/scala/Garage/MongoConnection.scala

```diff
@@ -0,0 +1,66 @@
+package Garage
+
+import org.mongodb.scala._
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.util.{Failure, Success}
+
+class MongoConnection {
+
+  val mongoClientPath = "//localhost"
+  val databaseName = "garage"
+
+  def getConnection(): MongoClient = {
+    val mongoClient = MongoClient("mongodb:" + mongoClientPath)
+    mongoClient
+  }
+
+  def closeCollection() = {
+    getConnection().close()
+  }
+
+  def getDatabase(): MongoDatabase = {
+    val database = getConnection().getDatabase(databaseName)
```

Comment:
why do you assign the value to variable and immediately return it, just return it in the first place